src/components/Carousel.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <!-- No need for this: hide-delimiter-background -->
+  <v-carousel
+    height="100%"
+    hide-delimiters
+    show-arrows-on-hover
+    @change="input"
+  >
+    <v-carousel-item
+      v-for="(word, id) in words"
+      :key="id"
+    >
+      <v-sheet
+        :color="colors[id % 5]"
+        height="100%"
+        @click="flip = !flip"
+      >
+        <v-row
+          class="fill-height"
+          align="center"
+          justify="center"
+        >
+          <div class="display-3" v-if="flip">{{ word[0] }}</div>
+          <div class="display-3" v-else>{{ word[1] }}</div>
+        </v-row>
+      </v-sheet>
+    </v-carousel-item>
+  </v-carousel>
+
+</template>
+
+<script>
+  import axios from "axios";
+  import { parse } from "papaparse";
+  export default {
+    async created () {
+      const { data } = await axios.get("/data.csv");
+      this.words  = parse(data).data.slice(1);
+    },
```

Comment:
I think including the json data straight into the `words` is fine for this example project. What do you think?